recipes/svtyper/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version="0.6.1" %}
+{% set version="0.7.0" %}
 
 package:
   name: svtyper
   version: {{ version }}
 
 source:
   url: https://github.com/hall-lab/svtyper/archive/v{{ version }}.tar.gz
-  md5: dddda640f93b1eb5426a39f0a392d6f4
+  md5: d7f298a5ba14f13d3d19af459e20d5b8
 
 build:
   number: 1
```

Comment:
@ernfrid, since you are updating the package to a new version, the build number should be 0. Please see "Build number and string", https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html, for more details.

Reply:
Done. Thank you for catching that and for the reference.